ja/orm/retrieving-data-and-resultsets.rst

```diff
@@ -1226,7 +1226,7 @@ reducer が呼ばれるごとに、reducer はユーザーごとのフォロワ
 
     $articles->find('commonWords')->mapReduce($mapper);
 
-stack されたすべての MapReduce 操作をを取り除く
+stack されたすべての MapReduce 操作を取り除く
 -----------------------------------------------
```

Comment:
Title underlines need to be the same length as the text above them.
Please delete 2 characters "-".